collections/_infographics/emissions/emissions-eu-trends.md

```diff
@@ -13,4 +13,38 @@ data-orig:
+* __Industrial processes:__ Emissions from industrial processes have fallen by 26% since 1990 to 343 million tonnes of CO<sub>2</sub>eq per year. These are emissions from processes that release greenhouse gases into the atmosphere other than through the combustion of fossil fuels - in the production of chemicals, cement, iron and steel, etc.  
+For example, emissions from cement production have fallen by almost a quarter since the 1990s. There has also been a decline in the chemical sector, with emissions from ammonia production falling by 34% and nitric acid production falling by 89% (they were still 46 Mt CO<sub>2</sub>eq in 1990). In contrast, there is a slight increase in emissions from petrochemicals. There are also decreases in iron and steel production (down 35%) and aluminium production (down 85%). Since the 1990s, emissions from the use of refrigerants have increased significantly, replacing the ozone-depleting substances previously used, which are now regulated under the Montreal Protocol. These emissions rose from zero to 88.4 Mt CO<sub>2</sub>eq in 2014. Since then, these same emissions have been declining, reaching 80.3 Mt CO<sub>2</sub>eq in 2019.
+* __Residential, institutional and agricultural combustion:__ Emissions fell by 27% compared to 1990 to 519 million tonnes of CO<sub>2</sub>eq per year. These emissions are from heating and hot water (unless a heating plant supplies energy), cooking, etc. It also includes fuel combustion in agriculture and forestry.
+* __Combustion in industry and construction:__ Emissions fell by 41% to 429 million tonnes of CO<sub>2</sub>eq in 2019. Emissions come from the combustion of fossil fuels for industrial use, typically when something needs to be heated in production (during melting, drying, distillation, etc.).  
+For example, emissions from combustion in heavy industry (iron and steel production) have halved by 2019.
+* __Agriculture:__ Emissions from the agricultural sector have fallen by 21% since 1990 to 382 million tonnes of CO<sub>2</sub>eq per year. Emissions come mainly from livestock farming and soil cultivation and associated N<sub>2</sub>O emissions.  
+Reducing methane emissions from agriculture would lead to a reduction in the number of livestock kept (and the associated reduction in the consumption of beef and dairy products), a change in the management of manure (for example, by stabilising it in biogas plants) and less intensive fertiliser application. However, reducing livestock farming can also harm soil quality, availability of natural fertiliser, etc.  
+* __Waste management:__ Emissions from waste management have been declining since the mid-1990s. By 2019, they had fallen by 34% to 114 million tonnes of CO<sub>2</sub>eq per year. They are mainly produced by landfills, which leak methane into the atmosphere. This methane is produced by decomposing biodegradable material (paper, cardboard, textiles and bio-waste) in the landfill.
```

Comment:
change:
Emissions from waste management have been declining since the mid-1990s. By 2019, they had fallen by 34% to 114 million tonnes of CO<sub>2</sub>eq per year. They are mainly produced by landfills, which leak methane into the atmosphere. This methane is produced by decomposing biodegradable material (paper, cardboard, textiles and bio-waste) in the landfill.
to
Emissions from waste management have been declining since the mid-1990s. By 2019, they had fallen by 34% to 114 million tonnes of CO<sub>2</sub>eq per year. These emissions are mainly produced by landfills, which leak methane into the atmosphere. This methane is produced by decomposing biodegradable material (paper, cardboard, textiles and bio-waste) in the landfill.

Reply:
Thanks, I've updated the second sentence as per suggestion: "These emissions are mainly produced..."